consultation_analyser/support_console/views/consultations.py

```diff
@@ -34,6 +37,20 @@ def index(request: HttpRequest) -> HttpResponse:
                 user = request.user
                 consultation.users.add(user)
                 messages.success(request, "A dummy consultation has been generated")
+            elif request.POST.get("generate_giant_dummy_consultation") is not None:
+                n = 10000
```

Comment:
should we put some checking here to make sure this is only happening in the local environment?

Reply:
There is a check in the `create_dummy_consultation_from_yaml`, and the buttons are only displayed in non-prod environments so I think fine (keen to run on dev not just local).